qa/os/src/test/java/org/elasticsearch/packaging/util/Archives.java

```diff
@@ -413,4 +418,59 @@ private static Path getPowershellOutputPath(Installation installation) {
         return installation.logs.resolve("output.out");
     }
 
+    public static void verifySecurityAutoConfigured(Installation es) throws Exception {
```

Comment:
Given this lives in `Archives` I guess this means we're never verifying these assertions for Docker installations? Should we be?

Reply:
There was the implicit ( admittedly more lightweight ) check on each docker test that we can connect to it over TLS while trusting the autoconfigured HTTP CA certificate. I'll mote these PackagingTestCase and verify the installation in Docker too after we run the container in `DockerTests.test010Install`